kwarchart/src/main/java/com/kwarchart/android/model/BarSeries.kt

```diff
@@ -0,0 +1,39 @@
+package com.kwarchart.android.model
+
+import androidx.compose.ui.graphics.Color
+import com.kwarchart.android.enum.BarChartType
+
+/**
+ * LineChart data.
+ */
+data class BarSeries<T>(
+        /**
```

Comment:
apply proper spacing
/** pls align it to its closing tag

Reply:
Let me handle this one... Will fix it in HORIZONTAL BarChart PR.